volatility3/framework/plugins/windows/drivermodule.py

```diff
@@ -0,0 +1,60 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+from volatility3.framework import renderers, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.renderers import format_hints
+from volatility3.plugins.windows import ssdt, driverscan
+
+# built in Windows-components that trigger false positives
+KNOWN_DRIVERS = ["ACPI_HAL",
+                "PnpManager",
+                "RAW",
+                "WMIxWDM",
+                "Win32k",
+                "Fs_Rec"]
+
+class DriverModule(interfaces.plugins.PluginInterface):
+    """Determines if any loaded drivers were hidden by a rootkit"""
+
+    _required_framework_version = (2, 0, 0)
+
+    @classmethod
+    def get_requirements(cls):
```

Comment:
```suggestion
    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
```

Reply:
I don't think many of these have their definition, but I'm ok putting it in...